package.json

```diff
@@ -51,7 +51,7 @@
   "devDependencies": {
     "@semantic-release/changelog": "6.0.3",
     "@semantic-release/git": "10.0.1",
-    "eslint": "8.56.0",
+    "eslint": "8.57.0",
```

Comment:
Updated ESLint version to use `loadESLint` function.

[ESLint v8.57.0 released - ESLint - Pluggable JavaScript Linter](https://eslint.org/blog/2024/02/eslint-v8.57.0-released/#new-api-loadeslint())